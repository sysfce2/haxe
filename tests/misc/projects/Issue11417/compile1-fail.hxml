Main1
-D message.reporting=pretty
-D message.no-color
