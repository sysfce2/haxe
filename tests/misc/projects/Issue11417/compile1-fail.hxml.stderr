[ERROR] Main1.hx:2: characters 28-29

 2 |  var foo = function(i:Int, i:Int) {};
   |                            ^
   | Duplicate argument name "i"

[ERROR] Main1.hx:3: characters 22-23

 3 |  function bar(i:Int, i:Int) {}
   |                      ^
   | Duplicate argument name "i"

[ERROR] Main1.hx:4: characters 30-31

 4 |  function bar1(i:Int, j:Int, i:Int) {}
   |                              ^
   | Duplicate argument name "i"

[ERROR] Main1.hx:5: characters 24-25

 5 |  function bar2(i:Int, ?i:Int) {}
   |                        ^
   | Duplicate argument name "i"

[ERROR] Main1.hx:6: characters 23-24

 6 |  function bar3(i:Int, i = 42) {}
   |                       ^
   | Duplicate argument name "i"

[ERROR] Main1.hx:7: characters 19-20

 7 |  var lambda = (i, i) -> {};
   |                   ^
   | Duplicate argument name "i"

